Documentation/contributing/docs/docstest.rst

```diff
@@ -12,11 +12,48 @@ Documentation Testing
 First, start a local document server that automatically refreshes when you save files for
 real-time preview. It relies on the ``cilium/docs-builder`` Docker container.
 
+Set up your development environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. Note::
+    To run Cilium's documentation locally, you need to install `docker engine <https://docs.docker.com/engine/install/>`_ and also the ``make`` package (Installed by default on Ubuntu).
+    To verify that ``make`` and ``docker`` is installed, run the command ``make --version`` and ``docker --version`` in your terminal.
```

Comment:
It's tempting to add fancy note sections to highlight specific pieces of the documentation, but these should be used sparingly. The documentation inside these notes stands by itself just directly in the page, there's no need for a callout box.

```suggestion
To run Cilium's documentation locally, you need to install `docker engine <https://docs.docker.com/engine/install/>`_ and also the ``make`` package (Installed by default on Ubuntu).

To verify that ``make`` and ``docker`` is installed, run the command ``make --version`` and ``docker --version`` in your terminal.
```

Reply:
Thank you. sorted them out now.